recipes/iqtree/build.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
+if [ "$(uname)" == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
Okay, as a quick sanity check, can you temporarily comment out this line, and the line in the recipe that applies the patch? I just want to make sure that forcing a 10.9 target is something necessary for this changeset.

Reply:
Builds failed. Adding the patch and deployment target back in